### PR TITLE
Release Issues

### PR DESCRIPTION
**Pull request by Prasanthkota165**
Solving Github issues

### CODE REVIEW

**Review by rishithrao57 — APPROVED** [review]

File: include/sudoku.h [hunk 1]
```diff
@@ -192,3 +216,36 @@ void col_groups(int column_group[][9], int b_1[9][3], int b_2[9][3], int b_3[9][
...
+#ifndef INPUT_SUDOKU_H
+#define INPUT_SUDOKU_H
+/** 
+ * @brief This function removes surplus characters or symbols from the input buffer.
+ *        Otherwise, if restricted symbols or characters have been entered during the
+ *        call to fgets(), the surplus characters (after MAX_DIGITS chars) remain in the input buffer
+ *        and will be wrongly accepted as input on the next iteration of the loop.
+ *
+ */
+static inline void ClearInputBuffer();
```
Comment (Mathieufalardeau3) [inline include/sudoku.h]: The functions should be named with lower case letters and spaces. The prof will deduct points for wrong standard.

File: include/sudoku.h [hunk 2]
```diff
@@ -192,3 +216,36 @@ void col_groups(int column_group[][9], int b_1[9][3], int b_2[9][3], int b_3[9][
...
+
+/** 
+ * @brief This function initially clears the input buffer and checks for restricted input values and returns only if the entered value 
+ *        is integer. 
+ *
+ * @param[in] *inputInteger --- This parameter takes the user input.
+ * 
+ */
+void getIntegerFromStdin(int *inputInteger);
```
Comment (Mathieufalardeau3) [inline include/sudoku.h]: The functions should be named with lower case letters and spaces. The prof will deduct points for wrong standard.

File: src/input_sudoku.c [hunk 3]
```diff
@@ -1,50 +1,105 @@
...
+
+	char c = 0; 
+	while ((c = getchar()) != '\n' && c != EOF); /*Loop over input buffer and consume chars until buffer is empty*/
+}
+
+/**
+ * This function initially clears the input buffer and checks for restricted 
+ * input values and returns only if the entered value is integer. 
+ */
+void getIntegerFromStdin(int *inputInteger){
```
Comment (Mathieufalardeau3) [inline src/input_sudoku.c]: The functions and variables should be named with lower case letters and spaces. The prof will deduct points for wrong standard.

**Review by gabriels2018 — APPROVED** [review]
I approve merging these changes

**Review by Mathieufalardeau3 — CHANGES_REQUESTED** [review]

**Review by Mathieufalardeau3 — APPROVED** [review]

**Comment by Prasanthkota165** [comment]
Looks good to me. Shall I merge it?

**Comment by rishithrao57** [comment]
Yes, do it.

**Pull request merged**